src/modules/Messages/MessagesController.js

```diff
@@ -10,10 +10,16 @@ class MessagesController extends Component {
     // when MessagesController mounts
     app.service('messages')
       .on('created', (message) => {
-        this.props.addMessage(message);
+        this.addMessage(message)
       });
   }
 
+  addMessage = (message) => {
+    this.props.addMessage(message);
+    const alert = new Audio(require('./assets/new-message.wav'));
```

Comment:
@ericgold @bbenefield89, in case you guys were wondering how to add sounds...  pretty simple...

I found the sound for new message here: https://freesound.org/people/KIZILSUNGUR/sounds/72125/

Feel free to suggest a different sound for new message.  @JaCorbett, do you want to find a sound that works for incoming call?